docs/api/screen.md

```diff
@@ -2,6 +2,9 @@
 
 > Retrieve information about screen size, displays, cursor position, etc.
 
+Process: [Main](../tutorial/quick-start.md#main-process),
+[Renderer](../tutorial/quick-start.md#renderer-process)
```

Comment:
Super duper minor, but looks like most places in the diff put both processes on a single line, but this one (and `shell.md) split it over two lines.